lib/utilities/find-build-file.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+var findUp = require('findup-sync');
+var path = require('path');
+
+module.exports = function(file) {
+  var buildFilePath = findUp(file); 
+
+  if (buildFilePath === null) {
+    throw new Error(file + ' not found');
```

Comment:
Throwing here means that we do not allow fallback to `Brocfile.js` with deprecation, can we have a test that has just a `Brocfile.js` and confirms that this still works?